medicaltorch/datasets.py

```diff
@@ -269,14 +274,16 @@ def _load_filenames(self):
                 self.indexes.append(item)
 
     def set_transform(self, transform):
-        """This method will replace the current transformation for the
+        """ This method will replace the current transformation for the
         dataset.
 
         :param transform: the new transformation
         """
         self.transform = transform
 
     def compute_mean_std(self, verbose=False):
+        # TODO: adapt to multi channels
```

Comment:
At this point we are always in mono-channel, no? Am I missing something?

Reply:
Oups you're right, forgot to change that! Mostly because this method is never called...  
Should we still adapt it ? if yes should we compute the mean/ std per channel or just for the entire dataset ??

Reply:
We should maybe use it... it could help (normalize with mean and std of the contrast-dataset).
Per channel makes more sense I think

Reply:
Fixed in [bc1dbbb](https://github.com/neuropoly/medicaltorch/pull/9/commits/bc1dbbbdbf2f61f0795496ee2e13266820848b00).

Reply:
I have seen it --> really really nice!